test/e2e/e2e.go

```diff
@@ -32,11 +32,16 @@ import (
 
 	"k8s.io/kubernetes/pkg/api"
 	"k8s.io/kubernetes/pkg/api/v1"
+	clientset "k8s.io/kubernetes/pkg/client/clientset_generated/release_1_5"
 	gcecloud "k8s.io/kubernetes/pkg/cloudprovider/providers/gce"
+	"k8s.io/kubernetes/pkg/runtime"
```

Comment:
hmm looks like these imports are ordered funny?  but not breaking any known rule so its ok i guess.